addons/spreadsheet_account/static/src/index.js

```diff
@@ -21,11 +21,29 @@ cellMenuRegistry.add("move_lines_see_records", {
         const sheetId = position.sheetId;
         const cell = env.model.getters.getCell(position);
         const { args } = getFirstAccountFunction(cell.compiledFormula.tokens);
-        let [codes, date_range, offset, companyId, includeUnposted] = args
-            .map(astToFormula)
-            .map((arg) => env.model.getters.evaluateFormulaResult(sheetId, arg));
-        codes = toString(codes?.value).split(",");
+        let codes, partner_ids = "";
+        let date_range, offset, companyId, includeUnposted = false;
+        const parsed_args = args.map(astToFormula).map(
+            (arg) => env.model.getters.evaluateFormulaResult(sheetId, arg)
+        );
+        if ( cell.content.startsWith("=ODOO.PARTNER.BALANCE") ) {
+            [partner_ids, codes, date_range, offset, companyId, includeUnposted] = parsed_args;
+        } else {
+            [codes, date_range, offset, companyId, includeUnposted] = parsed_args;
+        }
+        try {
+            codes = toString(codes?.value).split(",");
+        } catch {
+            codes = [];
+        }
         const locale = env.model.getters.getLocale();
+        if ( 
+            ( cell.content.startsWith("=ODOO.PARTNER.BALANCE")
+              || cell.content.startsWith("=ODOO.RESIDUAL")
+            ) && !date_range?.value
+        ) {
+            date_range = { value: new Date().getFullYear() }
```

Comment:
Why the current year? Isn't it supposed to be without boundaries for the partner balance?